cli/clistat/container.go

```diff
@@ -0,0 +1,61 @@
+package clistat
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+
+	"github.com/spf13/afero"
+	"golang.org/x/xerrors"
+)
+
+const (
+	procMounts    = "/proc/mounts"
+	procOneCgroup = "/proc/1/cgroup"
+)
+
+// IsContainerized returns whether the host is containerized.
+// This is adapted from https://github.com/elastic/go-sysinfo/tree/main/providers/linux/container.go#L31
+// with modifications to support Sysbox containers.
+// On non-Linux platforms, it always returns false.
+func IsContainerized(fs afero.Fs) (ok bool, err error) {
+	cgData, err := afero.ReadFile(fs, procOneCgroup)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return false, nil // how?
```

Comment:
what is the meaning of the comment `// how?`? This path would be taken if not in a cgroup?

Reply:
That comment is not valid and is probably copy-pasta, will remove.